website/docs/reference/deploy/environment-import-export.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Environment Import & Export
+---
+
+:::info Availability
+
+The environment import and export first appeared in Unleash 4.22.0.
+
+:::
+
+The environment export/import functionality in Unleash simplifies the process of moving feature flags between multiple instances. 
+It also makes it easier to move feature flags within a single instance between environments.
+When exporting you always select the toggles and environment to export from. When importing you always select a project and an environment to import to.
+
+## Import & Export items
+
+The following items will be exported and imported:
+* feature toggles
+* activation strategies
+* variants
+* enabled/disabled statuses
+* context fields
+* feature tags
+* feature tag types
+* segments
```

Comment:
I think we could clarify already at this point that exporting features also exports global reference items etc.

we should also add doc links for as many items as we can on this list (I've not done that in the suggestion)

Here's is a rough suggestion for what I think would be good here. What do you think?

```suggestion
When you export features, the export will contain both feature-specific configuration and global configuration.

On the project-level these  items are exported:

* the feature itself
* feature tags

On the environment-level, these items are exported for the chosen environment:

* activation strategies including constraints and references to segments
* variants
* enabled/disabled

Additionally, these global configuration items are exported:
* custom context fields 
* feature tag types

Importantly, while references to segments are exported, the segments themselves are **not** exported. Consult the [import rejection](#import-rejection) section for more information.
```

Reply:
I like the extra clarity that it brings.